scripts/misc/gen-esbuild-test.mjs

```diff
@@ -517,7 +517,7 @@ function ensureTreeSitterWasmGo() {
   fsExtra.ensureDirSync(path.dirname(TREE_SITTER_WASM_GO_FILENAME))
   return new Promise((rsl, rej) => {
     nodeHttps.get(
-      'https://github.com/tree-sitter/tree-sitter.github.io/raw/master/tree-sitter-go.wasm',
+      'https://tree-sitter.github.io/tree-sitter-go.wasm',
```

Comment:
I tried `wasm` from the previous url but got some runtime error, so I used `wasm` URL from the `tree-sitter` playground.